control-plane/tests/test_service_persistence_action_reconciliation.py

```
@@ -3594,13 +3594,131 @@ def test_service_inspects_action_review_states_on_queue_alert_and_case_surfaces(
+            review_owner="analyst-001",
+            intended_outcome="Review repository owner change evidence before any approval-bound response.",
+            lead_id=lead.lead_id,
+        )
+        action_request = service.create_reviewed_action_request_from_advisory(
+            record_family="recommendation",
+            record_id=recommendation.recommendation_id,
+            requester_identity="analyst-001",
+            recipient_identity="repo-owner-indexed-001",
+            message_intent="Notify the accountable repository owner about the reviewed permission change.",
+            escalation_reason="Reviewed GitHub audit evidence requires bounded owner notification.",
+            expires_at=datetime.now(timezone.utc) + timedelta(hours=4),
```

Ruff (0.15.10)

[error] Undefined name `datetime`

(F821)